scripts/setup_buckets.sh

```diff
@@ -0,0 +1,17 @@
+# Create bucket for resources in chosen region
+if aws s3 ls "s3://$2" 2>&1 | grep -q 'An error occurred.'
+then
+    aws s3api create-bucket --bucket $2 --region $1 --create-bucket-configuration LocationConstraint=$1
+    echo "Bucket '$2' has been created."
+else
+    echo "Bucket '$2' already exists."
+fi
```

Comment:
I know we also did this. But you can also create script to create a bucket and parse the bucket name as parameter. 
Than you do not have this duplication for every bucket

Reply:
😲🤦‍♀️🙈